src/time.cr

```diff
@@ -53,21 +53,55 @@ require "crystal/system/time"
 # span.hours # => 1
 # ```
 struct Time
-  # *Heavily* inspired by Mono's DateTime class:
-  # https://github.com/mono/mono/blob/master/mcs/class/corlib/System/DateTime.cs
-
   include Comparable(self)
 
-  TicksMask       =      0x3fffffffffffffff
-  KindMask        =      0xc000000000000000
-  MAX_VALUE_TICKS = 3155378975999999999_i64
+  # :nodoc:
+  DAYS_MONTH = [0, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
```

Comment:
Just a side node: Couldn't these constants be tuples?

Reply:
Thanks, I changed it